tests/categorical_test.py

```diff
@@ -107,18 +109,16 @@ def testStartsWith(self):
 
     def testGroup(self):
         group = self._getRandomizedCategorical().group()
-        self.assertTrue((ak.array([2, 5, 9, 6, 1, 3, 7, 0, 4, 8]) == group).all())
+        self.assertListEqual([2, 5, 9, 6, 1, 3, 7, 0, 4, 8], group.to_list())
 
     def testUnique(self):
         cat = self._getRandomizedCategorical()
 
-        self.assertTrue(
-            (
-                ak.Categorical(
-                    ak.array(["non-string", "string3", "string1", "non-string2", "string"])
-                ).to_ndarray()
-                == cat.unique().to_ndarray()
-            ).all()
+        self.assertListEqual(
+            ak.Categorical(ak.array(["non-string", "string3", "string1", "non-string2", "string"]))
+            .to_ndarray()
+            .tolist(),
```

Comment:
didn't i see a `to_list()` method for `Categorical`

Reply:
good catch! fixed!